opensrp-chw-core/src/main/java/org/smartregister/chw/core/dao/VisitDao.java

```diff
@@ -105,7 +105,7 @@ public static boolean memberHasVaccineCard(String baseEntityID) {
     }
 
     public static boolean memberHasVisits(String baseEntityID) {
-        String sql = "select count(*) total from visits where base_entity_id = '" + baseEntityID + "'";
+        String sql = "select count(*) total from visits where base_entity_id = '" + baseEntityID + "' AND visit_type NOT LIKE 'ANC Home Visit Not Done%'";
```

Comment:
This Returns Malaria Visits + All Fp Follow Up vists (From Registration Events / Change Methods / follow up visits)  + Pnc Visits. 
Is that the intended Result?